### ISSUE TITLE
robots.txt Bug Report

### ISSUE THREAD

**Issue by Binnar81** [Binnar81/MERNForm#2]
Here is a list of potential issues with the given code:

1. The code is missing a file extension. A robots.txt file should have a `.txt` extension. The correct code should look like this:
```makefile
User-agent: *
Disallow:
```
2. The `User-agent` field is set to `*`, which means the rules apply to all web crawlers. However, the `Disallow` field is empty, which means all parts of the website are allowed to be crawled. If the intention is to disallow all crawling, the `Disallow` field should be set to `/`:
```
User-agent: *
Disallow: /
```
3. The code is missing proper formatting. Although this might not be a functional issue, it is a good practice to format the code correctly for readability. The correct formatting should look like this:
```makefile
User-agent: *
Disallow: /
```

In summary, the given code is missing a file extension, has an incorrect `Disallow` field, and is not properly formatted. The corrected code should look like this:
```makefile
User-agent: *
Disallow: /
```
